docs/setup.md

```diff
@@ -0,0 +1,14 @@
+# ORCID auth instructions
+
+#### Steps to obtain ORCID
```

Comment:
Okk I would remove them and add steps to obtain new secret variable.
On Mar 16, 2016 1:44 AM, "Abigail Cabunoc Mayes" <notifications@github.com>
wrote:

> In docs/setup.md
> <https://github.com/mozillascience/PaperBadger/pull/172#discussion_r56232438>
> :
>
> > @@ -0,0 +1,14 @@
> > +# ORCID auth instructions
> > +
> > +#### Steps to obtain ORCID
>
> So those shouldn't be there! Could you look into removing that?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/mozillascience/PaperBadger/pull/172/files/248d4e4ab866665976f611119f9a395302594fa8#r56232438>
>